frontend/src/components/Player.vue

```diff
@@ -3,7 +3,12 @@
     <div class="info">
       <div class="name-info">
         <!-- <div class="avatar"><img src="../assets/icons/..." alt="avatar"></div> -->
-        <div class="name title-font">{{ player.name }}</div>
+        <div class="name title-font">
+          {{ player.name }}
+        </div>
+        <div v-if="winner" class="name winner">
+          <AwardIcon size="18" />
+        </div>
```

Comment:
`.name-info` uses `flex-direction: colum` for left and right players. If we added the `AwardIcon` component _inside_ the `.name` div, I think we should be fine, as this is using a regular `block` display. Something like this:

```html
<div class="name-info">
  <div class="name title-font">
    {{ player.name }}
    <AwardIcon size="18" v-if="winner">
  </div>
</div>
```

Reply:
oh, while we're at it: can we add a `title` attribute that adds a popover to the award icon with a small text that explains what the icon is supposed to mean? Just a small, i18n explanation like "this player wins the current trick" or something like that.

Reply:
> 
> 
> `.name-info` uses `flex-direction: colum` for left and right players. If we added the `AwardIcon` component _inside_ the `.name` div, I think we should be fine, as this is using a regular `block` display. Something like this:
> 
> ```
> <div class="name-info">
>   <div class="name title-font">
>     {{ player.name }}
>     <AwardIcon size="18" v-if="winner">
>   </div>
> </div>
> ```
![Left_wrap](https://user-images.githubusercontent.com/12951672/103372071-5b945480-4ad1-11eb-91e2-08139d6f0641.png)

Using this will wrap the badge into the next line, aligned to the left side. This was what I had initially, but unfortunately it's not looking good :cry: That's why I implemented it this way. I prefer it in the centre, therefore I think we should stick to this until we change it to be the way we want to have it (unwrapped).

Reply:
> 
> 
> oh, while we're at it: can we add a `title` attribute that adds a popover to the award icon with a small text that explains what the icon is supposed to mean? Just a small, i18n explanation like "this player wins the current trick" or something like that.

I like this very much.
I don't know right now if there is an easy way to implement this. I'm used to have external plugins taking care of tooltips or popus. I'm not sure how they will blend into handheld and touch devices though.
I thought of using your playernames 'tooltip' of the trick component, but i was having some issues when implementing issue #35 (pull request coming). I'd prefer not implementing this right now and instead we should discuss how to handle tooltips in general, since this will be something we will be relying heavily I suppose.

The [Vuetify framework has a tooltip](https://v2.vuetifyjs.com/en/components/tooltips/).
(Quasar has one too)[https://quasar.dev/vue-components/tooltip]
And there is a [single component which seems reliable](https://github.com/Akryum/v-tooltip).

Reply:
> Using this will wrap the badge into the next line, aligned to the left side

Oh, right, with long names that's gonna happen. You can prevent line breaks using `white-space: nowrap` on the `.name` div.

> I'm used to have external plugins taking care of tooltips or popus.
I'd argue that we keep it simple and use the browser-native `title="something"` tooltips only for now. It's mostly an accessibility concern and while browser-native tooltips are not really flashy, they're easy to implement and help people who rely on them.

Reply:
All suggestions implemented. {now-wrap, within .name, i18n}. I believe we can merge this then 👍
I'll let you do this in case there are a few more things to deal with.